search/includes/classes/class-versioning.php

```diff
@@ -184,6 +184,26 @@ public function get_active_version_number( Indexable $indexable ) {
 		return $active_version['number'];
 	}
 
+	/**
+	 * Grab just the version number for the inactive version
+	 *
+	 * @param \ElasticPress\Indexable $indexable The Indexable to get the inactive version number for
+	 * @return int|WP_Error The current inactive version number
+	 */
+	public function get_inactive_version_number( Indexable $indexable ) {
+		$inactive_versions = $this->get_inactive_versions( $indexable );
+
+		if ( empty( $inactive_versions ) ) {
+			return new WP_Error( 'no-inactive-versions-found', 'No inactive versions.' );
+		}
+
+		if ( count( $inactive_versions ) > 1 ) {
```

Comment:
I'm wondering if this is a correct approach to handling the many inactive versions. Like, we're asking for any inactive version, so maybe it's forth returning just the first one, rather than bail?

Reply:
@rinatkhaziev That was my first approach initially...but I changed it to throw an error because if the user is requesting for an inactive version, I figured it'd be better to throw an error to ensure that that the user instead targets a specific inactive version. I'm happy go either or.

Reply:
@rebeccahum I think having a single inactive version returned has more usefulness - i.e. if I as a user want to clean up some inactives I'd rather have a first inactive deleted that have an error thrown at me without any clue.

An alternative scenario I think might be to report all the inactive versions in the error:

`Found following inactive versions: 2,4,5. Please specify which one you would like to delete`

Reply:
@rinatkhaziev Let's do the first scenario you've proposed. Feedback should be addressed now.